### DISCUSSION TITLE
Verification Email EXTREMELY DELAYED! UNUSABLE!

### DISCUSSION THREAD

**Discussion by pmatty2** [community/community#119422] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When trying to login to my account the verification Email is taking between 6-8 hours to be delivered to my inbox, I tried reseting the password but this email also took 8 hours to arrive therefore neither the codes nor links to change the password work. 

I'm assuming this is a bug with email processing times but it makes the product unusable. I can't access my code. 

I had to create a new account just to post this. Even logging into Support was impossible.

**Comment by sanity-metal** [comment]
having the same problems, i needed to create a new account with a Microsoft email to submit a ticket. How ridiculous

**Comment by ryankwondev** [comment]
Same problem with @horang-admin account. @github

**Comment by donerre2** [comment]
same with @donerre since yesterday morning

**Reply by donerre** [reply]
after +- 24hrs trying i just got a code 3 mins before it would expire.. dont wanna give hope, but maybe u should try again ppl!
